### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AtillaTahak**
Hello Reviwers 💯 

Use Ruby gems as a software packages system.
Install Ruby on Rails framework.
Understand Rails RESTful design and router.
Use controllers to handle requests and render empty views.
Understand Rails naming conventions.
Use params from browser requests in a safe way.
Use preprocessed HTML file with embedded Ruby code.
Use layouts and templates for shared content.
Use database migration files to maintain database schema.
Use validations for models.
Secure app from n+1 problems.
Understand what ORM is.
Write SQL queries with ActiveRecord.
Set up associations between models.
Build a web app that requires the user to log in.
Describe the difference between authorization and authentication.
Use Devise gem for authentication.
Understand how sessions and cookies can support authentication.
Limit access to web app resources based on authorization rules.

### CODE REVIEW

File: app/controllers/foods_controller.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+class FoodsController < ApplicationController
```
Comment (GzimAsani) [inline app/controllers/foods_controller.rb:1]: - [ ] I can create new food without logging in, but when saving I get an error. I should not be able to do that
Screenshot from 2022-04-30 17-12-07

File: app/controllers/inventories_controller.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# rubocop:disable Lint/RescueException
+
+class InventoriesController < ApplicationController
```
Comment (GzimAsani) [inline app/controllers/inventories_controller.rb:3]: - [ ] Change styling to this ubtton, users cant really see it
Screenshot from 2022-04-30 17-21-13

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hi  team
 
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Your code organization is good.
 
 ## Required Changes 
 - Please explain better(in more detail) the steps that a user should take in order to clone and run your project




 _Check the comments under the review._
 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/foods_controller.rb (hunk 1 above)
Comment (cwaku) [reply app/controllers/foods_controller.rb:1]: Thank you for the review. Changes have been made. Logged out users are no more able to add food

File: app/views/devise/registrations/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```
Comment (FrederickMih) [inline app/views/devise/registrations/new.html.erb:34]: - [Optional] The appearance and design aspect of your pages is not all the best. I was unable to find the different signup fields in the signup form. You may improve on this. 


Screenshot from 2022-04-30 17-34-28

**Review by FrederickMih — APPROVED** [review]
Hi Team, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: app/views/devise/registrations/new.html.erb (hunk 3 above)
Comment (cwaku) [reply app/views/devise/registrations/new.html.erb:34]: Sure Thank you, we will do that

**Pull request merged**